### ISSUE TITLE
[Bug] Recommended Feed No Longer Working

### ISSUE THREAD

**Issue by roryyamm** [iv-org/invidious#4415]
**Describe the bug**
The 'Recommended' list, which usually goes beside the comments on a video, is no longer shown on any instance. 

**Steps to Reproduce**
1) Open Video
2) Attempt to find additional videos to watch
3) There are no other videos to watch
4) Feel freedom for the first time ever
5) Immediately want to escape this freedom

**Logs**
No logs. 

**Screenshots**
![screenshot_2024-01-31_at_00:54:56](https://github.com/iv-org/invidious/assets/24758288/d0ed8f50-c69d-48b6-a513-201c2f965077)
_A LazerPig Video on Yewtu.be, showing the distinct lack of recommended videos_

![screenshot_2024-01-31_at_00:58:17](https://github.com/iv-org/invidious/assets/24758288/b2cd61ab-e846-40d7-a1ab-6d37a4a3bab6)
_The same video on youtube.owacon.moe, showing even weird offshoots are affected by the bug_
![screenshot_2024-01-31_at_01:01:57](https://github.com/iv-org/invidious/assets/24758288/65f43910-5652-439e-926f-f2cb855c8854)
_A different video on Yewtu.be, one that was on the Trending page and thus would likely have recommendations turned on, showing no recommended videos_
**Additional context**
This also appears to affect Newpipe, which makes me think it's a backend change - someone altered the endpoint and now every recommended feed is broken except for on Youtube.

**Comment by shellheim** [comment]
I was just about to report this. Same in apps that use the Invidious api.

**Comment by ashley0143** [comment]
some videos just don't have any watch next videos, even on youtube (it's not in the response), like the video here.

Others will sometimes return a watch next response the other times the list is empty but has a continuation and fetching that continuation gives you the watch next videos. Don't know why it doesn't just include them in the initial response but presumably it's a bug on YouTube's side - it could take a long time for youtube to fix the initial response being empty, because it has no effect on them or the user experience of their website.
Their website automatically fetches the continuation when the bottom of the list becomes visible and as there are zero items to display, the bottom is visible on page load so it automatically fetches the continuation.

basically the fix would be just fetching the continuation when the watch next feed is empty but has a continuation - i dont know the language invidious'is written in so i cant do a thing lol

**Comment by ashley0143** [comment]
can confirm

**Comment by Orko79** [comment]
Same issue here. I´m using FreeTube on my deskto pc and pointed it to my selfhosted invidious instance. Recommended feed is not showing up in FreeTube and Invidious using the browser.

**Comment by rikafuruda** [comment]
Can confirm this, Currently happening on my instance as well.

**Comment by syeopite** [comment]
Should be fixed by #4416. Please let me know how well it works!

**Comment by ashley0143** [comment]
youtube fixed the issue :3
